path_rotate_root.go

```diff
@@ -8,8 +8,8 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/google/uuid"
```

Comment:
Any reason for this switch in uuid library?

Reply:
Don't need this after latest refactors of internal types and methods 👍🏼